src/index.ts

```diff
@@ -41,130 +53,39 @@ async function generateCMASpawnArguments(command) {
  * @returns {Promise<Object>} cumulusMessageAdapterObject - Returns an Object with a
  * childprocess and a stderror buffer
  * @returns {Object} cumulusMessageAdapterObject.cmaProcess - A CMA childProcess Object
- * @returns {string} cumulusMessageAdapterObject.errorObj -  A Object with the property
+ * @returns {Object} cumulusMessageAdapterObject.errorObj -  A Object with the property
  *                                                           'stderrBuffer' to make the encapsulated
  *                                                           error event storage outside this method
  */
-async function invokeCumulusMessageAdapter() {
+async function invokeCumulusMessageAdapter(): Promise<invokeCumulusMessageAdapterType> {
```

Comment:
Consider `InvokeCumulusMessageAdapterResult`. Types should start with a capital letter, and `Type` at the end should be avoided (based on best practices I've read).

Reply:
3ad90c2